httpql/src/main/java/com/hubspot/httpql/internal/BoundFilterEntry.java

```diff
@@ -15,9 +15,10 @@
 import com.hubspot.httpql.QuerySpec;
 import com.hubspot.httpql.ann.FilterBy;
 import com.hubspot.httpql.ann.FilterJoin;
+import com.hubspot.httpql.ann.desc.JoinDescriptor;
 import com.hubspot.httpql.impl.DefaultFieldFactory;
 
-public class BoundFilterEntry<T extends QuerySpec> extends FilterEntry {
+public class BoundFilterEntry<T extends QuerySpec>extends FilterEntry {
```

Comment:
did you mean to remove that space?

Reply:
Nope... some weird auto formatting going on. Will fix.